pkgs/tools/networking/tinystatus/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenvNoCC
+, makeWrapper
+, netcat
+, curl
+, unixtools
+, coreutils
+, mktemp
+, findutils
+, gnugrep
+, fetchFromGitHub
+}:
+stdenvNoCC.mkDerivation rec {
+  buildInputs = [ makeWrapper ];
+  pname = "tinystatus";
+  version = "unstable-2021-07-09";
+  src = fetchFromGitHub {
+    owner = "bderenzo";
+    repo = "tinystatus";
+    rev = "fc128adf240261ac99ea3e3be8d65a92eda52a73";
+    sha256 = "sha256-FvQwibm6F10l9/U3RnNTGu+C2JjHOwbv62VxXAfI7/s=";
+  };
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 tinystatus $out/bin/tinystatus
+    wrapProgram $out/bin/tinystatus \
+      --set PATH "${lib.makeBinPath [curl netcat unixtools.ping coreutils mktemp findutils gnugrep]}"
```

Comment:
Have you tried running nixpkgs-fmt on this bit?

Reply:
nixpkgs-fmt produces no changes on this file.